docker/docs/builder/run.sh

```diff
@@ -25,6 +25,7 @@ done
 sed -i '/onBrokenMarkdownLinks:/ s/ignore/error/g' docusaurus.config.js
 
 if [[ $# -lt 1 ]] || [[ "$1" == "--"* ]]; then
+  export CI=true 
```

Comment:
I can't find a description for the ENV in the wild. Can you add a doc link in the comment?